src/components/Results/DetectedIssueResources.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Accordion, AccordionSummary, AccordionDetails, Typography } from '@material-ui/core';
+import { R4 } from '@ahryman40k/ts-fhir-types';
+import fhirpath from 'fhirpath';
+
+interface Props {
+  detectedIssue: R4.IDetectedIssue;
+}
+
+const DetectedIssueResources: React.FC<Props> = ({ detectedIssue }) => {
+  const guidanceResponseArray = fhirpath.evaluate(detectedIssue, 'contained.GuidanceResponse');
+
+  return (
+    <div>
+      {guidanceResponseArray.map((response: R4.IGuidanceResponse) => {
+        return (
+          <Accordion>
+            <AccordionSummary>
+              <Typography>
+                <div> {fhirpath.evaluate(response, 'resourceType')} </div>
+                <div> id = {fhirpath.evaluate(response, 'id')} </div>
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+              <Typography>
+                <div> reasonCode = {fhirpath.evaluate(response, 'reasonCode.coding.code')} </div>
+                <div> display = {fhirpath.evaluate(response, 'reasonCode.coding.display')} </div>
+              </Typography>
```

Comment:
This an HTML nesting error. It actually doesn't affect functionality, but is best practice to avoid these kinds of errors:

![image](https://user-images.githubusercontent.com/16297930/125802116-588bb77b-f500-44e2-876e-59e305f5cb81.png)


This error is occurring because MUI's `Typography` component generates a `<p>` tag in HTML, and an HTML `<div>` cannot appear as a child. I assume you added the `div`s so the reasonCode and display for example appear on different lines. 

Another way to accomplish this should be to put these two things inside of their own Typography components rather than nesting them both under the same Typography. This applies to both the AccordionSummary and AccordionDetails